ios/ItaliaApp/Info.plist

```diff
@@ -1,6 +1,138 @@
+	<key>NSFaceIDUsageDescription</key>
+	<string>Enable Face ID for a faster identification.</string>
+	<key>NSLocationAlwaysUsageDescription</key>
+	<string>The app needs NSLocationAlwaysUsageDescription permission</string>
+	<key>NSLocationUsageDescription</key>
+	<string>The app needs NSLocationUsageDescription permission</string>
+	<key>NSLocationWhenInUseUsageDescription</key>
+	<string>The app needs NSLocationWhenInUseUsageDescription permission</string>
+	<key>NSMicrophoneUsageDescription</key>
+	<string>IO needs access to the microphone in case you want to leave a voice note</string>
+	<key>NSMotionUsageDescription</key>
+	<string>The app needs NSMotionUsageDescription permission</string>
+	<key>NSPhotoLibraryAddUsageDescription</key>
+	<string>The app needs NSPhotoLibraryAddUsageDescription permission</string>
+	<key>NSPhotoLibraryUsageDescription</key>
+	<string>IO needs access to the Photos to scan QR codes</string>
+	<key>NSSpeechRecognitionUsageDescription</key>
+	<string>The app needs NSSpeechRecognitionUsageDescription permission</string>
+	<key>UIAppFonts</key>
+	<array>
+		<string>TitilliumWeb-Black.ttf</string>
+		<string>TitilliumWeb-Bold.ttf</string>
+		<string>TitilliumWeb-BoldItalic.ttf</string>
+		<string>TitilliumWeb-ExtraLight.ttf</string>
+		<string>TitilliumWeb-ExtraLightItalic.ttf</string>
+		<string>TitilliumWeb-Italic.ttf</string>
+		<string>TitilliumWeb-Light.ttf</string>
+		<string>TitilliumWeb-LightItalic.ttf</string>
+		<string>TitilliumWeb-Regular.ttf</string>
+		<string>TitilliumWeb-SemiBold.ttf</string>
+		<string>TitilliumWeb-SemiBoldItalic.ttf</string>
+		<string>Ionicons.ttf</string>
+		<string>RobotoMono-Bold.ttf</string>
+		<string>RobotoMono-BoldItalic.ttf</string>
+		<string>RobotoMono-Light.ttf</string>
+		<string>RobotoMono-LightItalic.ttf</string>
+		<string>RobotoMono-Regular.ttf</string>
+		<string>RobotoMono-RegularItalic.ttf</string>
+		<string>io-icon-font.ttf</string>
+	</array>
+	<key>UIBackgroundModes</key>
+	<array>
+		<string>remote-notification</string>
+	</array>
+	<key>UILaunchStoryboardName</key>
+	<string>LaunchScreen</string>
+	<key>UIRequiredDeviceCapabilities</key>
+	<array>
+		<string>armv7</string>
+	</array>
+	<key>UISupportedInterfaceOrientations</key>
+	<array>
+		<string>UIInterfaceOrientationPortrait</string>
+	</array>
+	<key>UIViewControllerBasedStatusBarAppearance</key>
+	<false/>
+	<key>com.apple.developer.nfc.readersession.iso7816.select-identifiers</key>
+	<array>
+		<string>com.apple.developer.nfc.readersession.iso7816.select-identifiers</string>
+		<string>A0000000308000000009816001</string>
+		<string>A00000000039</string>
+		<string>A0000002471001</string>
+		<string>00000000000000</string>
+	</array>
+	<key>method</key>
+	<string>app-store</string>
+	<key>NFCReaderUsageDescription</key>
+	<string>NFC tag to read NDEF messages into the application</string>
+</dict>
+</plist>
+=======
```

Comment:
Maybe this is a merge conflict

Reply:
Yep I was fixing it! Fixes on commit 482f0d6